pkgs/development/libraries/cxx-rs/default.nix

```diff
@@ -0,0 +1,53 @@
+{ cxx-rs, fetchFromGitHub, lib, rustPlatform, testers }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cxx-rs";
+  version = "1.0.72";
+
+  src = fetchFromGitHub {
+    owner = "dtolnay";
+    repo = "cxx";
+    rev = version;
+    sha256 = "sha256-+OumeLSgz8kLQKhEc3icCk1q+X+S7Xt+XtAlkx8iguU=";
+  };
+
+  cargoPatches = [
```

Comment:
If you're doing this, might as well just put the Cargo.lock in-tree and remove cargoSha256.